packages/alert-message-react/CHANGELOG.md

```diff
@@ -7,17 +7,17 @@ See [Conventional Commits](https://conventionalcommits.org) for commit guideline
 
 ### Bug Fixes
 
-- la ikonet endre størrelse i compact-variant ([6c0c5aa](https://github.com/fremtind/jokul/commit/6c0c5aaaba5b868d21c97d4b3e0a0212a2ef6f44))
+-   la ikonet endre størrelse i compact-variant ([6c0c5aa](https://github.com/fremtind/jokul/commit/6c0c5aaaba5b868d21c97d4b3e0a0212a2ef6f44))
 
 ### Features
 
-- bump majorversjon av core ([0b25b9c](https://github.com/fremtind/jokul/commit/0b25b9ccb4d35214037e45158264fab2da196a5f))
-- ta i bruk de nye ikonene i AlertMessage ([de50d55](https://github.com/fremtind/jokul/commit/de50d5503356467e2d99de894a6a77f215bbf827))
+-   bump majorversjon av core ([0b25b9c](https://github.com/fremtind/jokul/commit/0b25b9ccb4d35214037e45158264fab2da196a5f))
```

Comment:
Alle disse innsrykksendringene i CHANGELOG-filer trenger kanskje ikke å være med. De blir jo uansett overskrevet når Lerna versjonerer før publisering.

Reply:
De er snarveien min for å få en diff tagget breaking i React-pakkene :P